scripts/run-integration-tests.sh

```diff
@@ -133,7 +104,8 @@ start-daemons() {
 		--libp2p-port=9955 \
 		--rpc-port 5002 \
 		"--bootnodes=${ALICE_MULTIADDR}" \
-		"--contract-address=${SWAP_FACTORY_ADDR}"
+		"--contract-address=${SWAP_FACTORY_ADDR}" \
+		"--claim-relayer"
```

Comment:
wait so bob always uses a relayer now? don't we want to still test non-relayer txs?

Reply:
That flag `--claim-relayer` here is on Charlie's `swapd` instance.  It is not Bob or Alice's `swapd`. So Charlie is the instance providing relaying services in the integration tests.

Bob will only use the relayer in these cases, which I don't think have changed (yet) in this PR:
* No ETH to pay for the claim
*  He uses `--relayer-fee` from `swapcli` or, equivalently, passes a non-nil value in the `RelayerFee` field of `MakeOfferRequest`.

What I'd like to change based on other PR feedback:
* Remove the `RelayerFee` entirely from the `MakeOffer` request and replace it, at least temporarily, with a boolean `--force-relayer` (or `--use-relayer`) flag. This will allow our integration tests to work without a major overhaul. It will be nice even for manually testing, as you won't have to clear out your ETH balance to exercise the relayer.

My vision is that Alice will be the relay over 90% of the time. If we come up with ideas to reliably make Alice the relayer 100% of the time when Bob needs it, it will probably be solid gold for reducing code/protocol/end-user complexity. Bob's life is easier when he only needs to negotiate with one party instead of two.  :)

Reply:
I don't think we can ever guarantee someone will be able to find a relayer unfortunately, people might choose not to relay for whatever reason. 

I think a `--use-relayer` flag instead of `--relayer-fee` is fine